app/test_main.py

```diff
@@ -1 +1,46 @@
 # write your code here
+import pytest
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_url, internet_connection, access",
+    [
+        pytest.param(
+            True,
+            True,
+            "Accessible",
+            id="url valid and there is connection"
+        ),
+        pytest.param(
+            True,
+            False,
+            "Not accessible",
+            id="url valid, but there is no connection"
+        ),
+        pytest.param(
+            False,
+            True,
+            "Not accessible",
+            id="url isn't valid, but this is connection"
+        ),
+        pytest.param(
+            False,
+            False,
+            "Not accessible",
+            id="url isn't valid and there is no connection"
+        )
+    ],
+)
+def test_can_access_google_page(
+        valid_url: str,
```

Comment:
`valid_url` is a boolean value, change the type annotation for it to encounter the underlying type.